sources/packages/backend/workflow/test/test-utils/mock/supporting-user-information-request/check-supporting-user-response-task-mock.ts

```diff
@@ -0,0 +1,27 @@
+import { WorkflowSubprocesses, WorkflowServiceTasks } from "../..";
+import { createMockedWorkerResult } from "../mock.utils";
+
+/**
+ * Creates the mock for 'Check Supporting User Response' completed task.
+ * @param options mock options.
+ * - `totalIncome` total income that must be present to allow the workflow
+ * to advance.
+ * - `subprocesses` subprocess reference when the workflow was invoked
+ * using a call activity. It can be defines as one or multiple ones case
+ * the workflow was invoked from subprocess inside another subprocess.
+ * @returns mock for 'Check Supporting User Response' completed task.
+ */
+export function createCheckSupportingUserResponseTaskMock(options: {
+  totalIncome: number;
+  subprocesses?: WorkflowSubprocesses;
+}): Record<string, unknown> {
+  return createMockedWorkerResult(
+    WorkflowServiceTasks.CheckSupportingUserResponseTask,
+    {
+      jobCompleteMock: {
+        totalIncome: options.totalIncome,
```

Comment:
Is the totalIncome required to be returned for the test case to run ?

In real this worker will return either of these right? `parent1TotalIncome|parent2TotalIncome|partner1TotalIncome`